app/javascript/components/side-panel.jsx

```diff
@@ -4,13 +4,15 @@ import { spacing } from '@lib/theme';
 import FixedContent from '@components/fixed-content';
 import widgetShape from '@lib/widget-shape';
 
+export const width = '600px';
```

Comment:
Sorry, I wasn't clear before. This can remain an int. You would just need to modify the styles in Wrapper to be `${width}px;` which would mean you don't need the `parseInt`'s further down